### ISSUE TITLE
Touch dragging of GridItem doesn't work as expected on mobile Android devices

### ISSUE THREAD

**Issue by pbabey** [jbaysolutions/vue-grid-layout#234]
Hi gmsa,

first i'd like to thank you for this great component. I use it since early 2017.
I've migrated my project to the newest version 2.2.0. Everything works fine, except the mobile experience on Android. The touch behavior on Android, isn't as good as on iOS devices. On Android, I can't drag around the GridItems. On iOS devices it's smooth and works properly. 
I'd like to create a PR, but i can't get the project work locally. Do I need to do something special?

Many thanks in advance

**Comment by gmsa** [comment]
Just run `npm install` and `npm run serve` from the project root

**Comment by pbabey** [comment]
This is was I did initially. When I execute the command `npm run serve` I get following error: Cannot find module 'webpack/lib/RequestShortener'

**Comment by gmsa** [comment]
Just tested a fresh install and worked fine. Makre sure your node and npm versions are updated and try deleting the node_modules folder and package-lock.json

**Comment by pbabey** [comment]
Thank's after deleting deleting the node_modules folder and package-lock.json it's working. Do you have a hint, where to look into to fix the touch issue?

**Comment by gmsa** [comment]
Maybe checking if its a problem that comes from the library used for dragging and resizing: http://interactjs.io/
If it is, there's not much that can be done about it, other than maybe switching to another lib.

**Comment by onx2** [comment]
@pbabey 
Adding this CSS removes all touch support. If it is only broken on Android devices, perhaps another solution should be considering via JavaScript? 

Testing on a Windows 10 Dell XPS15 touch seems to work fine for dragging and resizing when I remove this CSS. 

```
 //  Removes touch support on all laptops, tablets, and mobile devices
touch-action: none;
```

**Comment by pbabey** [comment]
@onx2 
I found a open issue regarding this bug on InteractJs (https://github.com/taye/interact.js/issues/595). 
Thank's for your javascript snippet.  I will create a new PR, to only remove the css class when it's on a android.

**Comment by onx2** [comment]
No problem @pbabey, happy to contribute! Yes, they should fix it there as well, but there was a pull request a few days ago (#235) that broke touch for all devices (not just Android). Removing ```touch-action: none;``` from the grid item CSS will fix it for most devices. I would be better to support touch on as many devices as possible because either way touch will be broken on that one Android device. 

Any chance we can get this fix in so people wont need to run manual updates to keep touch support / stay up-to-date with this software? Or can you elaborate on the interact issue and how their fix can override the CSS that was added?

**Comment by gmsa** [comment]
Released 2.3.1 with a fix for this. The CSS is only applied for Android browsers. Thanks for the help @onx2 !
